Airports/EGPD/Ownership.txt

```diff
@@ -1,39 +1,39 @@
 SECTOR:Only PDGND 1:0:0
-OWNER:PDA:PDF:PDT:PDG
+OWNER:PDR:PDF:PDT:PDG
 
 SECTOR:Only PDGND 2:0:0
-OWNER:PDA:PDF:PDT:SS:SE:S:SM:O
+OWNER:PDR:PDF:PDT:SS:SE:S:SM:O
 
 SECTOR:Only PDTWR 1:0:0
-OWNER:PDA:PDF:PDT
+OWNER:PDR:PDF:PDT
 
 SECTOR:Only PDTWR 2:0:0
-OWNER:PDA:PDF:SS:SE:S:SM:O
+OWNER:PDR:PDF:SS:SE:S:SM:O
 
 SECTOR:Only PDAPP:0:0
-OWNER:PDA
+OWNER:PDR
 
 SECTOR:PDTWR:0:2000
-OWNER:PDT:PDA:PDF:SS:SE:S
-ALTOWNER:Observing Scottish FIR:O:SM:PDT:PDA:PDF:SS:SE:S
-ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PDT:PDA:PDF:SS:SE:S
-ALTOWNER:Mentoring TWR/APP:PDTM:PDT:PDAM:PDA:PDF:SS:SE:S
+OWNER:PDT:PDR:PDF:SS:SE:S
+ALTOWNER:Observing Scottish FIR:O:SM:PDT:PDR:PDF:SS:SE:S
+ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PDT:PDR:PDF:SS:SE:S
+ALTOWNER:Mentoring TWR/APP:PDTM:PDT:PDRM:PDR:PDF:SS:SE:S
 BORDER:PDTWR
 DEPAPT:EGPD
 ARRAPT:EGPD
 
 SECTOR:PDAPP:0:12000
-OWNER:PDA:PDF:SE:S
-ALTOWNER:Observing Scottish FIR:O:SM:PDA:PDF:SS:SE:S
-ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PDA:PDF:SS:SE:S
-ALTOWNER:Mentoring TWR/APP:PDAM:PDA:PDF:SS:SE:S
-BORDER:PDA_FL120
+OWNER:PDR:PDF:SE:S
+ALTOWNER:Observing Scottish FIR:O:SM:PDR:PDF:SS:SE:S
+ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PDR:PDF:SS:SE:S
+ALTOWNER:Mentoring TWR/APP:PDRM:PDR:PDF:SS:SE:S
+BORDER:PDR_FL120
```

Comment:
The SECTORLINE will also need to change name, can't see it changed here?

Reply:
accidental change corrected